src/Spectre.libGenetic/DataTypes.h

```diff
@@ -23,6 +23,7 @@ limitations under the License.
 namespace spectre::algorithm::genetic
 {
 using ScoreType = double;
+using Label = signed;
```

Comment:
It is specific to `libClassifier`. It should never be here. Everything that is dependent on this type should not appear in `libGenetic`.

Reply:
done